frontend/src/TooljetDatabase/Drawers/BulkUploadDrawer/FileDropzone.jsx

```diff
@@ -23,7 +23,13 @@ export function FileDropzone({
 
   const { getRootProps, getInputProps, isDragActive, acceptedFiles } = useDropzone({
     accept: { parsedFileType: ['text/csv'] },
-    onDrop,
+    onDrop: (acceptedFiles) => {
+      if (acceptedFiles.length > 0) {
```

Comment:
We will only deal with one file as of now right? I think we can make the code imply that for now?

```
acceptedFiles.length === 1
```